### ISSUE TITLE
nvda does not read the window name correctly in a java application

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#9184]
### Steps to reproduce:
Open a java application with multiple windows, each window must have a different name.
### Actual behavior:
Press nvda + t to have nvda read the name of the current window.
Nvda announces the name of the first window of the java application.
### Expected behavior:
Nvda should read the name of the current java application window, and should not repeat the name of the first open java window in the application.
#### NVDA installed/portable/running from source:
install
#### NVDA version:
alfa 16587 e 2018.4.1
#### Windows version:
10 16299
#### Name and version of other software in use when reproducing the issue:
The java application used is an application written within the company I work for.
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
yes
I tested with a version recommended by the organization where I work 2016.3 and it works correctly.

**Comment by DrSooom** [comment]
@fernando-jose-silva: Which Java SE Runtime Environment version is installed?

**Comment by fernando-jose-silva** [comment]
the java installed are the 64-bit and 32-bit versions (the 32-bit version was installed per end) version 8 update 201

**Comment by mwhapples** [comment]
Duplicate of #7571

**Comment by mwhapples** [comment]
I have made a pull request #13744 which should fix this amongst a number of other Java swing issues. You may want to test that pull request to see if it fixes this issue for you.

**Comment by fernando-jose-silva** [comment]
sorry now i can't test this attempt, but i can test it on monday by brazil time when i will be at my work again and i will have access to java application.

**Comment by fernando-jose-silva** [comment]
Perfect, this attempt at build 25490 solves this problem, anxiously awaiting the next steps, to go to alpha and then stable releases.
There is a user where I work who has been waiting for this fix since 2016, a user having difficulties using her computer, she was reluctant to leave nvda 2016.xx until earlier this year because of this unresolved issue.
